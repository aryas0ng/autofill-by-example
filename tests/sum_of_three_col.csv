1,0,1,2
2,4,2,8
2,3,3,8
3,8,4,
10,10,13,
5,5,1,
4,4,11,
19,4,13,
2,21,16,
1,21,16,
9,10,19,
7,7,2,
4,4,20,
3,3,3,
12,3,3,
4,16,3,
1,16,3,
26,27,4,
8,8,46,
3,3,5,
3,3,6,
8,11,6,
4,4,7,