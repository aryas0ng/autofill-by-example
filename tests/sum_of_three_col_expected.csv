1,0,1,2
2,4,2,8
2,3,3,8
3,8,4,15
10,10,13,33
5,5,1,11
4,4,11,19
19,4,13,36
2,21,16,39
1,21,16,38
9,10,19,38
7,7,2,16
4,4,20,28
3,3,3,9
12,3,3,18
4,16,3,23
1,16,3,20
26,27,4,57
8,8,46,62
3,3,5,11
3,3,6,12
8,11,6,25
4,4,7,15